src/core/parseHTML.js

```diff
@@ -21,14 +21,29 @@ jQuery.parseHTML = function( data, context, keepScripts ) {
 		context = false;
 	}
 
-	// Stop scripts or inline event handlers from being executed immediately
-	// by using document.implementation
-	context = context || ( support.createHTMLDocument ?
-		document.implementation.createHTMLDocument( "" ) :
-		document );
-
-	var parsed = rsingleTag.exec( data ),
-		scripts = !keepScripts && [];
+	var base, parsed, scripts;
+
+	if ( !context ) {
+
+		// Stop scripts or inline event handlers from being executed immediately
+		// by using document.implementation
+		if ( support.createHTMLDocument ) {
+			context = document.implementation.createHTMLDocument( "" );
+
+			// Support: Chrome <=46 - 48
```

Comment:
This is not just Chrome but also Firefox as it doesn't provide the resolved URL there. Also, we're really required to do it so I'm not sure if we can claim it's to support some browser as if it had a bug. It's just something we have to do to make it work as we want, the previous way was incorrect.